initialize.sh

```diff
@@ -71,9 +71,8 @@ else
     echo "Docker version $docker_version detected"
   fi
 fi
-
 # Check if docker-compose is installed
-if ! [ -x "$(command -v docker-compose)" ]; then
+if ! [ -x "$(find /usr | grep "docker-compose$")" ]; then
```

Comment:
why this changed?

Reply:
because docker-compose is stored in the cli-plugins dir in the usr dir(default:/usr/local/lib/docker/cli-plugins and in my case /usr/libexec/docker/cli-plugins/) but I should change this to work withe older version too

Reply:
I was wrong here sorry